packages/ag-charts-website/src/content/docs/fills/index.mdoc

```diff
@@ -51,9 +67,9 @@ The array should contain a minimum of two objects, with each object providing op
 -   `color`: Any valid CSS colour value.
 -   `stop`: A ratio between `0` and `1` indicating the position within the gradient where the colour changes.
 
-```js
-series: [
-    {
+```js format="snippet"
+{
+    series: {
```

Comment:
Is this a bug? Array to object

Reply:
Manually fixed, may have been broken on one of the many many iterations to get this working.